pkgs/development/libraries/sqlite/default.nix

```diff
@@ -12,19 +12,23 @@ in
 
 stdenv.mkDerivation rec {
   pname = "sqlite";
-  version = "3.35.2";
+  version = "3.35.5";
 
   # NB! Make sure to update ./tools.nix src (in the same directory).
   src = fetchurl {
     url = "https://sqlite.org/2021/sqlite-autoconf-${archiveVersion version}.tar.gz";
-    sha256 = "1bfczv5006ycwr1vw7xbq7cmys0jvfr8awmx7wi1b40zyj0yss8j";
+    sha256 = "9StypcMZw+UW7XqS4SMTmm6Hrwii3EPXdXck9hMubbA=";
   };
 
   outputs = [ "bin" "dev" "out" ];
   separateDebugInfo = stdenv.isLinux;
 
   buildInputs = [ zlib ] ++ optionals interactive [ readline ncurses ];
 
+  preConfigure = lib.optionalString (stdenv.isAarch64) ''
```

Comment:
```suggestion
  preConfigure = ''
```

Reply:
since this is going into staging, may as well apply the changes to all platforms.